verification/spec/connection-handshake/Environment.tla

```diff
@@ -0,0 +1,233 @@
+---------------------------- MODULE Environment ----------------------------
```

Comment:
Not ideal name. Maybe it can be connection handshake and the other connection end, or something along those lines.

Reply:
Igor proposed `Medium`. I'll think more about this.